frontend/components/adopter/ViewPets/components/AnimalListing.tsx

```diff
@@ -0,0 +1,87 @@
+import { Animal } from "@contract";
+import { Typography } from "antd";
+import styled from "styled-components";
+
+interface AnimalListingProps {
+	animal: Animal.Attributes;
+}
+
+const GenderDisplay = {
+	F: "Female",
+	M: "Male",
+};
+
+export function AnimalListing(props: AnimalListingProps) {
+	const {
+		animal: { name, gender, breed, weightKg, animalImages },
+	} = props;
+	const thumbnail = animalImages?.[0].thumbnailUrl;
```

Comment:
Later on, we might want to use a placeholder image to say no available image. Currently it looks like this which should be fine.
![Screenshot 2021-12-28 at 11 15 11 PM](https://user-images.githubusercontent.com/3167732/147580486-49d0f664-44b0-4d3a-b66e-cec4d9221a8f.png)